app/src/main/java/com/sun/moviedb_53/ui/detail/movie/MovieDetailFragment.kt

```diff
@@ -1,31 +1,56 @@
 package com.sun.moviedb_53.ui.detail.movie
 
+import android.annotation.SuppressLint
 import android.os.Bundle
 import android.util.DisplayMetrics
 import android.view.View
 import androidx.core.os.bundleOf
 import com.sun.moviedb_53.R
 import com.sun.moviedb_53.base.BaseFragment
+import com.sun.moviedb_53.data.model.MovieDetail
+import com.sun.moviedb_53.data.source.MovieRepository
+import com.sun.moviedb_53.data.source.local.MovieLocalDataSource
+import com.sun.moviedb_53.data.source.remote.MovieRemoteDataSource
+import com.sun.moviedb_53.extensions.loadFromUrl
+import com.sun.moviedb_53.utils.Constant
 import kotlinx.android.synthetic.main.fragment_movie_details.*
 import kotlin.math.roundToInt
 
-class MovieDetailFragment : BaseFragment() {
+class MovieDetailFragment : BaseFragment(), MovieDetailContact.View {
 
     private var idMovie: Int? = null
+    private var detailPresenter: MovieDetailPresenter? = null
 
     override fun getLayoutId() = R.layout.fragment_movie_details
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        detailPresenter = MovieDetailPresenter(
+            MovieRepository.getInstance(
+                MovieRemoteDataSource.getInstance(),
+                MovieLocalDataSource.getInstance()
+            )
+        )
         arguments?.let {
             idMovie = it.getInt(ID_MOVIE_DETAIL)
         }
+        detailPresenter?.let { detailPresenter_ ->
+            detailPresenter_.setView(this)
+            idMovie?.let { detailPresenter_.getMovieDetail(it) }
+        }
```

Comment:
```suggestion
        detailPresenter?.let { 
            it.setView(this)
            idMovie?.let { it.getMovieDetail(it)
             }
        }
```

Reply:
cái it ở dòng 40 nó k nhận đc anh ơi. nó báo đỏ getMovieDetail

Reply:
nên em sửa lại thành it1 luôn á a